src/jspdf.js

```diff
@@ -3026,7 +3026,7 @@ function jsPDF(options) {
             "<style>html, body { padding: 0; margin: 0; } iframe { width: 100%; height: 100%; border: 0;}  </style>" +
             '<body><iframe id="pdfViewer" src="' +
             pdfJsUrl +
-            '?file=" width="500px" height="400px" />' +
+            '?file=&downloadName=' + options.filename + '" width="500px" height="400px" />' +
```

Comment:
The arguments seem a little messed up now...

Reply:
This is the only way so far that I can think of to pass the file name for download using pdf.js. Currently, it defaults to "ducment.pdf". The arguments in viewer.js seem always has only one element - the blob URL, so it's not useful for that matter.

Reply:
Yeah, that's alright. It's just that the URL parameters now look like this
```
?file=&downloadName=filename.pdf
```
which looks a bit broken. Do you know what the "file=" part does? Seems to me like we can remove that part.

Reply:
Actually `file=` is required, it provides the PDF file for rendering. Normally it would be `?file=/folder/test.pdf`. Here it is blank and I think the pdf content is populated by bloburl instead. I tried to remove it, but `?downloadName=filename.pdf` returns error: 
![image](https://user-images.githubusercontent.com/21064307/90632980-3e7e7980-e1f3-11ea-84c6-83a67f74c690.png)

Reply:
Ah, ok. Didn't test it. Then everything is fine ;)

Reply:
Thanks! Just for the record, it passes the bloburl as the file name to pdf.js first, then replaces it with a physical pdf file name/path if exists. How the bloburl is inserted as the file name is not clear to me yet.